src/Parallelization.php

```diff
@@ -389,6 +389,7 @@ protected function executeMasterProcess(InputInterface $input, OutputInterface $
                     '--child',
                     '--env='.$input->getOption('env'),
                     $input->getOption('no-debug') ? '--no-debug' : '',
+                    $this->serializeInputOptions($input, ['child','env','no-debug', 'processes']),
```

Comment:
why is `no-debug` passed? If you pass it to the parent shouldn't it be passed to the children?

Reply:
Because no-debug is already passed in line 391.  Same for "--env".
We could remove lines 390 and 391 and then exclude those two parameters from the blacklist in line 391. Do you agree? Probably the blackList then only needs to contain ``processes``.